spec/models/manageiq/providers/azure/cloud_manager/refresher_spec.rb

```diff
@@ -200,7 +231,7 @@ def assert_specific_hardware_networks(v)
     network = v.hardware.networks.where(:description => "public").first
     expect(network).to have_attributes(
       :description => "public",
-      :ipaddress   => "40.76.199.78",
+      :ipaddress   => "40.121.149.27",
       :hostname    => "ipconfig1"
     )
     network = v.hardware.networks.where(:description => "private").first
```

Comment:
@djberg96 - might want to add a spec for the security_group property for hardware.networks (if we support that, see my comments above)